### ISSUE TITLE
does not work with AIRCO2NTROL COACH

### ISSUE THREAD

**Issue by gebi** [markuslindenberg/co2monitor_exporter#1]
Hi,

awesome someone made a golang implementation, i was just too far into the python implementation for a rewrite ( https://github.com/gebi/co2exporter ).

Though your exporter does not work with "TFA-Dostmann AIRCO2NTROL Coach CO2 Monitor".

It just outputs...

```
# ./co2monitor_exporter --device=/dev/hidraw6
INFO[0000] Starting co2monitor_exporter (version=, branch=, revision=)  source="co2monitor_exporter.go:32"
INFO[0000] Build context (go=go1.14.1, user=, date=)     source="co2monitor_exporter.go:33"
INFO[0000] Listening on :9673                            source="co2monitor_exporter.go:70"
FATA[0000] checksum error: 73 f2 1a 7c 43 b1 e8 32       source="co2monitor_exporter.go:57"
```

The same device works with my exporter
```
# ./co2exporter.py /dev/hidraw6
Listening on :9672, appending labels: {}
*42: 127D  4733    T: 22.66
*41: 0B8A  2954,  42: 127D  4733    T: 22.66 RH: 29.54
 41: 0B8A  2954,  42: 127D  4733, *53: 0000     0    T: 22.66 RH: 29.54
 41: 0B8A  2954,  42: 127D  4733, *50: 0307   775,  53: 0000     0    CO2:  775 T: 22.66 RH: 29.54
 41: 0B8A  2954, *42: 127D  4733,  50: 0307   775,  53: 0000     0    CO2:  775 T: 22.66 RH: 29.54
*41: 0B8A  2954,  42: 127D  4733,  50: 0307   775,  53: 0000     0    CO2:  775 T: 22.66 RH: 29.54
 41: 0B8A  2954,  42: 127D  4733,  50: 0307   775, *53: 0000     0    CO2:  775 T: 22.66 RH: 29.54
```

**hint** the Coach variant of the CO2 monitor does not "encrypt" it's data, so the strategy i'm using is to just check the checksum on the unencrypted data and if it matches use that, otherwise decrypt and try again to verify the checksum, eg. https://github.com/gebi/co2exporter/blob/master/co2exporter.py#L96

**Comment by markuslindenberg** [comment]
Thanks, that's interesting. I have the smaller version w/ «encryption». I expected the Coach to work the same, but also already got feedback that it doesn't work. I'll look into it and would love to support both out of the box but can't test because I don't have that hardware.

**Comment by gebi** [comment]
NP, i was equally surprised, because i don't have the smaller version but just the Coach and it required a bit of tinkering to get it to work (i liked the Coach because it has a nice UI even without something "smart" getting the values and displaying it).

I've even a test Coach HW here connected to my laptop, if you have a version ready i can test it without much hassl.

As i don't have the smaller version it would be nice if you could test my exporter on your HW (the smaller one) if it works :), thx!

**Comment by markuslindenberg** [comment]
I assume this was fixed by #3, please reopen if necessary. Thanks!
